concent_api/concent_api/system_check.py

```diff
@@ -0,0 +1,74 @@
+from django.core.checks     import Error, register
+from django.conf            import settings
+from django.core.validators import URLValidator
+from django.core.exceptions import ValidationError
+
+
+@register()
+def check_settings_concent_features(app_configs, **kwargs):  # pylint: disable=unused-argument
+    errors = []
+
+    concent_features = settings.CONCENT_FEATURES
```

Comment:
There is no need to create this extra variable, just use settings.CONCENT_FEATURES directly in code below.